src/components/client/core/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,55 @@
+import { Portal } from "@ark-ui/react";
+
+import {
+  Tooltip as PrimitiveTooltip,
+  TooltipArrow,
+  TooltipArrowTip,
+  TooltipContent,
+  TooltipPositioner,
+  TooltipTrigger,
+} from "components/primitives";
+import { tooltip } from "generated/panda/recipes";
+import { useIsMounted } from "lib/hooks";
+
+import type { TooltipProps } from "components/primitives";
+import type { ReactNode } from "react";
+
+export interface Props extends TooltipProps {
+  trigger: ReactNode;
+  content: ReactNode;
+}
+
+/**
+ * Core UI tooltip.
+ */
+const Tooltip = ({ trigger, content, ...rest }: Props) => {
+  const classNames = tooltip();
+
+  const isMounted = useIsMounted();
+
+  if (!isMounted) return null;
+
+  return (
+    <PrimitiveTooltip {...rest}>
```

Comment:
Only thing I think I would do is add zero open and close delay by default like so:

```ts
const Tooltip = ({
  trigger,
  content,
  openDelay = 0,
  closeDelay = 0,
  ...rest
}: Props) => {
  const classNames = tooltip();

  const isMounted = useIsMounted();

  if (!isMounted) return null;

  return (
    <PrimitiveTooltip openDelay={openDelay} closeDelay={closeDelay} {...rest}>
    ...
    ```
    
Can then update the example app demos (remove `openDelay` and `closeDelay`).